velox/docs/functions/spark/math.rst

```diff
@@ -14,12 +14,15 @@ Mathematical Functions
 .. spark:function:: ceil(x) -> [same as x]
 
     Returns ``x`` rounded up to the nearest integer.  
-    In spark only long, double, and decimal have ceil.
+    In Spark, Supported types are: BIGINT and DOUBLE.
```

Comment:
Drop "In Spark,". Say "Supported types are ..."

Reply:
Thanks, just fixed it.